test/src/test/java/org/corfudb/runtime/view/AbstractViewTest.java

```diff
@@ -172,6 +172,9 @@ public void cleanupBuffers() {
         testServerMap.values().forEach(x -> {
             x.getLogUnitServer().shutdown();
             x.getManagementServer().shutdown();
+            x.getLayoutServer().shutdown();
```

Comment:
please rename `x` to `server`

P.S. never use one letter variable, it's hard to read

Reply:
Done.